### PR TITLE
Add confirmation email to nominee form

### PR DESCRIPTION
**Pull request by rgoj**
~~[WORK IN PROGRESS]~~ Ready for review!

[Favro card](https://favro.com/organization/b03d6d6d9b11b61167ac4246/30e671746b81b504ef983ee8?card=Dev-8536) (see the individual subcards underneath this one for the details of the changes made.

This PR modifies the nomination form (Get Involved -> Nominate on the site), mainly:
- adds a confirmation email field
- simplifies what is collected about the nominee

### CODE REVIEW

File: apps/addresses/migrations/0005_auto_20190124_1328.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```
Comment (rgoj) [inline apps/addresses/migrations/0005_auto_20190124_1328.py]: There are three migration files in this PR, the the two named `000X_auto_XXXX` are not related - probably old unapplied migrations - they got created when I ran `makemigrations`

File: apps/people/forms.py [hunk 2]
```diff
@@ -2,23 +2,47 @@
...
 from django.forms import inlineformset_factory
+from django.core.exceptions import ValidationError
 
 from . import choices as persons_choices
 from .models import Person, Nominee
 
 
 class NominateForm(forms.ModelForm):
     required_css_class = 'required'
+    confirm_email = forms.EmailField(label="Re-enter email address:")
```
Comment (richardsmithdevsoc) [inline apps/people/forms.py]: single quotes

File: apps/people/views.py [hunk 3]
```diff
@@ -40,24 +35,21 @@ def post(self, request, *args, **kwargs):
 
         form = self.get_form(form_class)
         nominee_form = NominatorFormSet(request.POST)
-        address_form = AddressForm(request.POST)
-        if (form.is_valid() and nominee_form.is_valid() and address_form.is_valid()):
-            return self.form_valid(form, nominee_form, address_form)
+        if (form.is_valid() and nominee_form.is_valid()):
```
Comment (richardsmithdevsoc) [inline apps/people/views.py]: I think you can get rid of the outer parentheses here.

File: apps/people/views.py [hunk 4]
```diff
@@ -40,24 +35,21 @@ def post(self, request, *args, **kwargs):
...
+            return self.form_valid(form, nominee_form)
         nominee_form.is_valid()
```
Comment (richardsmithdevsoc) [inline apps/people/views.py]: This works fine, one small thing, it looks like `nominee_form.is_valid().is_valid()` is being called twice now, should only need to validate the form once, could maybe do something like:
```
nominee_form = NominatorFormSet(request.POST)
nominee_form_is_valid = nominee_form.is_valid()
if form.is_valid() and nominee_form_is_valid:
    return self.form_valid(form, nominee_form)
```

Comment (rgoj) [reply apps/people/views.py]: Nice changes, thanks, implemented!

**Comment by rgoj** [comment]
@richardsmithdevsoc Thanks for your feedback, now implemented, re-assigning if you want to have a second look!

**Pull request merged**